pkg/registry/apis/provisioning/resources/replicator.go

```diff
@@ -365,15 +365,15 @@ func (r *Replicator) Export(ctx context.Context) error {
 		}
 
 		name := item.GetName()
-		logger := logger.With("item", name)
+		ctx, logger := slogctx.With(ctx, logger, "item", name)
```

Comment:
We should not overuse context fields with `slogctx.With` as they cluttered the logs. Normally you use only the bare minimum that helps that helps you understand the callstack. 

For example, in this case `item` contains the `folder` and `file_name` apart from a lot of other stuff that it's not relevant down the stack.